apps/yggdrasil/test/test_helper.exs

```diff
@@ -1,4 +1,8 @@
 ExUnit.start()
 
-Mix.Task.run "ecto.create", ~w(-r Yggdrasil.Repo --quiet)
-Mix.Task.run "ecto.migrate", ~w(-r Yggdrasil.Repo --quiet)
+repo = ~w(-r Yggdrasil.Repo --quiet)
+
+Mix.Task.run "ecto.drop", repo
+Mix.Task.run "ecto.create", repo
+Mix.Task.run "ecto.migrate", repo
+Mix.Task.run "run", ~w(apps/yggdrasil/priv/repo/seeds.exs)
```

Comment:
@copenhas so I needed the seeds to run, added it here and had it drop the db as well because it's reseeding the permissions each time.

Now that roles are being seeded we can't just simply migrate each time locally, it should be dropped. Doesn't effect travis obviously, but does locally.

That said, this also breaks local tests if you are in the yggdrasil sub app, so I had to keep toggling this file. I think we need to look at hoisting this out of the test_helper into a mix task or something.